src/services/user.service.js

```diff
@@ -0,0 +1,33 @@
+const { User } = require('../models/User.model');
+const jwtService = require('../services/jwt.service');
+
+const getByActivationToken = async (activationToken) => {
+  return User.findOne({ where: { activationToken } });
+};
+
+const getByEmail = async (email) => {
+  return User.findOne({ where: { email } });
+};
+
+const getById = async (id) => {
+  return User.findOne({ where: { id } });
+};
+
+const getByAccessToken = async (accessToken) => {
+  const { id } = await jwtService.verify(accessToken);
+  const user = await getById(id);
```

Comment:
Ensure that the 'id' extracted from the token is valid before attempting to retrieve the user. If 'id' is undefined, this will cause an error when calling getById.